src/pages/ClaimsPage/ClaimsPage.tsx

```diff
@@ -248,9 +268,7 @@ const ClaimRow = ({
       </td>
       <td>
         <Text>
-          {formatEpochDates(
-            claimsGroupByVault[distribution.vault.vault_address]
-          )}
+          {formatEpochDates(claimsGroupByRow[claimsRowKey(distribution)])}
```

Comment:
this would be a bit easier to read, maybe a tiny bit faster too, if you pushed the generation of `claimsGroupByRow[claimsRowKey(distribution)]` into a const above rather than running it twice.

actually, wait, you can just calculate it in the parent and pass it into ClaimRow as a prop, instead of passing in the entire `claimsGroupByRow`